cmd/up.go

```diff
@@ -158,6 +160,26 @@ List of components that kubeapps up installs:
 			return err
 		}
 		clientset, err := kubernetes.NewForConfig(config)
+		tries := 0
+		writer := uilive.New()
+		writer.Start()
```

Comment:
you should `defer writer.Stop()` here probably, not sure what the consequences of not calling Stop are if we return early.

Reply:
fixed